runner/operator/roslin_operator/construct_roslin_pair.py

```diff
@@ -67,7 +67,18 @@ def construct_roslin_jobs(samples):
         job['pair'] = [tumor_sample, normal_sample]
         references = convert_references(project_id, assay)
         job.update(references)
-        roslin_jobs.append(job)
+
+        job_metadata = {}
+        job_metadata['assay'] = assay
+        job_metadata['request_id'] = project_id
+        job_metadata['tumor'] = {}
+        job_metadata['tumor']['igo_id'] = tumor['igo_id']
```

Comment:
@allanbolipata @sivkovic which identifier are we using inside the Roslin pipelines? IGO ID or Patient ID or some other ID?